README.md

```diff
@@ -1,18 +1,39 @@
-### Title Bot
+# Title Bot
+## General goal
 
-Create a web form that allows a user to type in a url to any website and have it then output the Title of that website on the same page. For example, if I typed in CNN.COM into the text input of the web page, it would spit out:
+Create a web form that allows a user to type in a url to any website and have it then output the Title and favicon of that website on the same page. For example, if I typed in CNN.COM into the text input of the web page, it would spit out something like the following:
 
-> CNN - Breaking News, Latest News and Videos
+>![CNN Favicon](https://www.cnn.com/favicon.ie9.ico "CNN Favicon") | CNN - Breaking News, Latest News and Videos
 
 The application should consist of a javscript/html/css front end, and a backend API that fetches the page title.
+The web page should display a historical list of all urls entered by the user, along with their titles.
 
 You may use your preferred language for developing. Our preferred languages at chatmeter are scala and javascript.
 
 This task should take you somewhere around 4 hours. Make a git repository available online and send us the link. Make sure that the application can be run by us with minimal set up. You will present the application during your next interview.
 
-**Bonus points:**
-- Use React or any other front end web framework for the web form (Vue, Angular, Ember, etc.)
-- Write the API server in scala or node.js
-- Make it look great. Styling with Bootstrap is preferred
-- Display a historical list of all urls entered by the user, along with their titles
-- Dockerize the application
\ No newline at end of file
+## Requirements
+### Technical requirements
+- Use React for the front end
```

Comment:
I don't see too much of a problem keeping this part somewhat open-ended rather than strictly react